native/app/components/SplashPanel.js

```diff
@@ -19,7 +28,7 @@ class SplashPanel extends React.Component {
           actions: [NavigationActions.navigate({ routeName: 'LoginPanel' })],
         }));
       }
-    });
+    }).catch((e) => console.error(e));
```

Comment:
nit: if something fails here let's try do to a navigation to the login panel. instead of just dropping the error?